src/slack_to_notion/slack_client.py

```
@@ -150,6 +151,55 @@ def fetch_channel_info(self, channel_id: str) -> dict:
+        try:
+            response = self.client.users_info(user=user_id)
+            user = response["user"]
+            profile = user.get("profile", {})
+            name = (
+                profile.get("display_name")
+                or profile.get("real_name")
+                or user.get("real_name")
+                or user_id
+            )
+            self._user_cache[user_id] = name
+            return name
```

Ruff (0.15.1)

[warning] Consider moving this statement to an `else` block

(TRY300)